sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClient.java

```diff
@@ -56,6 +58,59 @@
         this.encryptedBlobAsyncClient = encryptedBlobAsyncClient;
     }
 
+    /**
+     * Creates and opens an output stream to write data to the block blob. If the blob already exists on the service, it
+     * will be overwritten.
+     *
+     * @return A {@link BlobOutputStream} object used to write data to the blob.
+     * @throws BlobStorageException If a storage service error occurred.
+     */
+    public BlobOutputStream getBlobOutputStream() {
+        return getBlobOutputStream(false);
+    }
+
+    /**
+     * Creates and opens an output stream to write data to the block blob. If the blob already exists on the service, it
```

Comment:
I don't think the second sentence is relevant here since we accept the overwrite flag on this overload.

Reply:
I'll change it here and on the block blob doc